specifications/api/swagger.yaml

```diff
@@ -2065,11 +2582,24 @@ x-responsesGetWalletMigrationInfo: &responsesGetWalletMigrationInfo
         schema: *ApiWalletMigrationInfo
 
 x-responsesMigrateWallet: &responsesMigrateWallet
-  <<: *responsesErr403
-  <<: *responsesErr404
-  <<: *responsesErr405
+  403:
+    description: Forbidden
+    content:
+      application/json:
+        schema:
+          oneOf:
+            - <<: *errNothingToMigrate
+            - <<: *errNoRootKey
+            - <<: *errWrongEncryptionPassphrase
+  <<: *responsesErr404WalletNotFound
   <<: *responsesErr406
-  <<: *responsesErr415
+  410:
+    description: Gone
+    content:
+      application/json:
+        schema:
+          <<: *errNoSuchWallet
```

Comment:
This doesn't feel right to me. I think we should use `404` on 'no such wallet'. Especially looking at some endpoints where:

![Screenshot from 2020-10-30 11-25-22](https://user-images.githubusercontent.com/42900201/97694299-a1697800-1aa2-11eb-8c4c-d16402df5a71.png)

Reply:
This PR isn't really about refactoring or fixing our error codes though. I just added documentation to the existing error handlers. I changed nothing.

Reply:
ACK. It revealed those small inconsistencies though (I spotted 410 and 403 only). 
Could be fixed in this go if it's not a lot of work, but don't have to of course.